src/databricks/labs/dqx/engine.py

```diff
@@ -590,6 +592,18 @@ def load_checks_from_installation(
             raise ValueError(f"Invalid or no checks in workspace file: {installation.install_folder()}/{filename}")
         return parsed_checks
 
+    def load_checks_from_table(self, table_name: str, query: str | None = None) -> list[dict]:
+        """
+        Load checks (dq rules) from a Delta table in the workspace.
+        :param table_name: Unity catalog or Hive metastore table name
+        :param query: Spark SQL expression to filter checks loaded from the table
+        :return: List of dq rules or raise an error if checks file is missing or is invalid.
+        """
+        if not self.ws.tables.exists(table_name).table_exists:
+            raise NotFound(f"Table {table_name} does not exist in the workspace")
+        logger.info(f"Loading quality rules (checks) from table {table_name}")
```

Comment:
You could log this before checking if the table exists so that the context is clear.